frontend/src/router/pages/SubmitDates.jsx

```diff
@@ -68,18 +92,40 @@ function SubmitDates() {
 
     const response = await sendData(payload);
 
+    if (savingDraft) {
+      setNotes("");
+      fetchData();
+      openFlashMessage(
+        "Dates saved as draft",
+        `${season?.park.name} ${season?.featureType.name} ${season?.operatingYear} season details saved`,
```

Comment:
Same note as above with the optional chaining. We could use some kind of conditional check to make sure `season` is set, or otherwise show some other string... but I don't think that case can ever happen. Maybe it's safe to take out the `?`s all together?

Reply:
Same here, season is just the data for the whole page. So, it can happen that season is null. But it could never happen season is not null and park is null. 
I'm not a huge fan of using `?` anyway. Maybe at one point, it'd be better to just do all the checks at the beginning and not worry about optional chaining operators.

Reply:
Even with the optional chaining, you'd end up with a sentence with missing words. In most cases in the template, it's easier and safer to just wrap it in something like `{season && (<blah blah>)}` so it won't even render until the data is loaded.

Reply:
All good! I learned something new about optional chaining today 😅 🥴